Cargo.toml

```diff
@@ -16,5 +16,4 @@ futures = "0.1"
 tokio-core = "0.1"
 tokio-io = "0.1"
 bytes = "0.4"
-mio = "0.6"
-mio-serial = "0.5"
+mio-serial = { git = "https://github.com/berkowski/mio-serial" }
```

Comment:
Please change to `mio-serial = { git = "https://github.com/berkowski/mio-serial", branch = "serialport-2"` }